soweego/wikidata/api_requests.py

```diff
@@ -357,6 +357,9 @@ def _lookup_label(item_value):
     if not response_body:
         LOGGER.warning('Failed label lookup for %s', item_value)
         return None
+    if not response_body.get('entities'):
+        LOGGER.warning('No entities for %s', item_value)
+        return None
```

Comment:
We should actually refactor this, because it looks like duplicate code.
Why don't we reuse `_sanity_check`?

Reply:
I'll look into it, but you know the validator way better than me.

Reply:
Don't worry, I'm gonna fix this.